lib/counting.cc

```diff
@@ -511,11 +511,19 @@ CountingHashFileReader::CountingHashFileReader(
         unsigned int save_ksize = 0;
         unsigned char save_n_tables = 0;
         unsigned long long save_tablesize = 0;
+        char signature [14];
         unsigned char version = 0, ht_type = 0, use_bigcount = 0;
 
+        infile.read(signature, 14);
         infile.read((char *) &version, 1);
         infile.read((char *) &ht_type, 1);
-        if (!(version == SAVED_FORMAT_VERSION)) {
+        if (!(std::string(signature) == SAVED_COUNTGRAPH_SIGNATURE)) {
+            std::ostringstream err;
+            err << "Does not start with signature for an Oxli " <<
```

Comment:
I'm OK with oxli and CountGraph being used internally, but it seems problematic to put them in diagnostic/error output, no?